### DISCUSSION TITLE
Hiding the key in a KeyValue field

### DISCUSSION THREAD

**Discussion by nettsite** [filamentphp/filament#6982] [Help] [answered]
Hi,

I want to allow users to add a variable number of email addresses into a single object e.g. a customer, etc. I will store it in the database as a JSON column. The KeyValue field seems to be the best bet, but I need to show only the "Value" bit on the screen - the key can be numeric or anything else behind the scenes.

Is there any way to achieve this?

I would use a textarea or something as simple, except that I want to validate each email address.

**Comment by nettsite** [comment]
Thanks, that would work, but repeaters each get thier own fieldset, and take up way too much space on the screen, and just look clumsy. 

Maybe I could fiddle with the CSS to reduce spacing, hide borders, etc, I guess.

**Reply by pboivin** [reply, marked as answer]
May I suggest https://github.com/awcodes/filament-table-repeater
It's much more compact visually than the built-in repeater.

**Reply by nettsite** [reply]
That looks good, thank you!

Just tried it, absolutely perfect.

**Reply by pege-dev** [reply]
this not working for v4 stable
